src/DevBetterWeb.Web/Pages/Admin/SubscriptionPlans/Index.cshtml.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.EntityFrameworkCore;
+using DevBetterWeb.Core.Entities;
+using DevBetterWeb.Infrastructure.Data;
+using Microsoft.AspNetCore.Authorization;
+using DevBetterWeb.Core;
+
+namespace DevBetterWeb.Web.Pages.Admin.SubscriptionPlans
+{
+  [Authorize(Roles = AuthConstants.Roles.ADMINISTRATORS)]
+  public class IndexModel : PageModel
+    {
+        private readonly AppDbContext _context;
+
+        public IndexModel(AppDbContext context)
+        {
+            _context = context;
+        }
+
+        public IList<MemberSubscriptionPlan>? MemberSubscriptionPlansList { get;set; }
+
+        public async Task OnGetAsync()
+        {
+            MemberSubscriptionPlansList = await _context.MemberSubscriptionPlan!.AsQueryable().ToListAsync();
+        }
+    }
+}
```

Comment:
See if you can fix some of the warnings in the GitHub build. They seem to mostly be possible null value dereferences and async methods missing await keywords. Otherwise looks good.